addons/l10n_be/data/template/account.account-be.csv

```diff
@@ -22,91 +22,110 @@
-"a230000","Plant, Machinery and Equipment - Acquisition Value","230000","asset_fixed","","False","","Installations, machines et outillage - Valeur d'acquisition","Installaties, machines en uitrusting - Aanschaffingswaarde","Anlagen, Maschinen und Betriebsausstattung - Anschaffungswert","Installations, machines and equipment directly related to your business activities","Installations, machines et outillage directement liés à vos activités professionnelles","Installaties, machines en uitrusting rechtstreeks verbonden aan je bedrijfsactiviteiten","Anlagen, Maschinen und Betriebsausstattung, die in direktem Zusammenhang mit Ihrer Geschäftstätigkeit stehen","attn_VAT-IN-V83-21","asset_machinery"
-"a230008","Plant, Machinery and Equipment - Capital Profits","230008","asset_fixed","","False","","Installations, machines et outillage - Plus-values","Installaties, machines en uitrusting - Meerwaarden","Anlagen, Maschinen und Betriebsausstattung - Mehrwerte","","","","","",""
-"a230009","Plant, Machinery and Equipment - Depreciations or Write-Downs (-)","230009","asset_fixed","","False","","Installations, machines et outillage - Amortissements ou réductions de valeur (-)","Installaties, machines en uitrusting - Afschrijvingen of waardeverminderingen (-)","Anlagen, Maschinen und Betriebsausstattung - Abschreibungen oder Wertminderungen (-)","","","","","",""
-"a240000","Office Furniture and Equipment - Acquisition Value","240000","asset_fixed","","False","","Mobilier et matériel de bureau","Kantoormeubilair en -uitrusting - Aanschaffingswaarde","Büromöbel und -ausstattung - Anschaffungswert","Large expenses related to workplaces","Dépenses importantes liées aux lieux de travail","Grote uitgaven verbonden aan werkplekken","Hohe Ausgaben im Zusammenhang mit Arbeitsplätzen","attn_VAT-IN-V83-21","asset_office"
-"a240008","Office Furniture and Equipment  - Capital Profits","240008","asset_fixed","","False","","Mobilier et matériel de bureau - Plus-values","Kantoormeubilair en -uitrusting - Meerwaarden","Büromöbel und -ausstattung - Mehrwerte","","","","","",""
-"a240009","Office Furniture and Equipment  - Depreciations or Write-Downs (-)","240009","asset_fixed","","False","","Mobilier et matériel de bureau - Amortissements ou réductions de valeur (-)","Kantoormeubilair en -uitrusting - Afschrijvingen of waardeverminderingen (-)","Büromöbel und -ausstattung - Abschreibungen oder Wertminderungen (-)","","","","","",""
+"a215000","Software - Acquisition Value","215000","asset_fixed","","False","","Logiciels - Valeur d'acquisition","Software - Aanschaffingswaarde","Software - Anschaffungswert","Software bought for your business activities (ERP, CRM)
+& operating systems : Windows, macOS, Linux...","Logiciels achetés pour vos activités professionnelles","Software gekocht voor je bedrijfsactiviteiten","Für Ihre Geschäftstätigkeit erworbene Software","attn_VAT-IN-V83-21","asset_software"
```

Comment:
New line ?! :monocle_face: Is it intended, and why ? 
 (other occurrences below)